scripts/spec-check.py

```
@@ -59,14 +70,19 @@ def main():
         idx = text.find("\n---\n")
         if idx >= 0:
             text = text[idx + 1:]
-
-    if not text.startswith("---\n"):
-        fail(["缺 YAML frontmatter（必须以 --- 开头）"])
-    parts = text[4:].split("\n---", 1)
-    if len(parts) < 2:
-        fail(["frontmatter 未闭合（缺结束 ---）"])
+        else:
+            fail(["缺 YAML frontmatter（必须以 --- 开头）"])
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)